src/pages/MatchList/MatchList.tsx

```diff
@@ -1,32 +1,54 @@
+import { useMemo } from 'react'
 import { useNavigate } from 'react-router-dom'
+import { useMatches } from '../../queries/match'
 import { Match } from '../../types/Match'
-import { useOwnMatches } from '../../hooks/useOwnMatches'
+import { getErrorMessage } from '../../utils/api'
+import { CREATE_MATCH } from '../../routes/routeSpecs'
 
 import { motion } from 'framer-motion'
-
-import { Fab, List, Box } from '@mui/material'
+import { Fab, List, Box, Typography } from '@mui/material'
 import { MatchCard } from '../../components/MatchCard/MatchCard'
 import AddIcon from '@mui/icons-material/Add'
 import { animationProps } from '../../styles/animation'
 import { styledFloatButton } from '../../styles/floatingButton'
-import { CREATE_MATCH } from '../../routes/routeSpecs'
+import { Alert } from '../../components/Alert'
+import CircularProgress from '@mui/material/CircularProgress'
 
 export const MatchList = () => {
   const navigate = useNavigate()
-  const loadedMatches = useOwnMatches()
+  const { data, isError, error, isLoading } = useMatches()
+
+  const matches = useMemo(
```

Comment:
We might not need to memoize this, since the only state that can cause a re-render in this component is the match fetching itself. In other words, this `useMemo` will always be re-calculated when the component re-renders, since it depends on the only state that will be changing